src/main/java/org/symphonyoss/integration/webhook/github/parser/v2/GithubCommentMetadataParser.java

```diff
@@ -0,0 +1,129 @@
+import static org.symphonyoss.integration.webhook.github.GithubEventTags.ISSUE_TAG;
+import static org.symphonyoss.integration.webhook.github.GithubEventTags.USER_TAG;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import org.symphonyoss.integration.model.yaml.IntegrationProperties;
+import org.symphonyoss.integration.parser.ParserUtils;
+import org.symphonyoss.integration.parser.SafeString;
+import org.symphonyoss.integration.service.UserService;
+import org.symphonyoss.integration.webhook.github.parser.GithubParserUtils;
+
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * This class is responsible to validate the event 'commit_comment' sent by Github Webhook when
+ * the Agent version is equal to or greater than '1.46.0'.
+ * Created by apimentel on 10/05/17.
+ */
+@Component
+public class GithubCommentMetadataParser extends GithubMetadataParser {
+
+  private static final String METADATA_FILE = "metadataGithubComment.xml";
+
+  private static final String TEMPLATE_FILE = "templateGithubComment.xml";
+  private static final String COMMIT_ENTITY = "commit";
+  private static final String ISSUE_ENTITY = "issue";
+
+  @Autowired
+  public GithubCommentMetadataParser(UserService userService, GithubParserUtils utils,
+      IntegrationProperties integrationProperties) {
+    super(userService, utils, integrationProperties);
+  }
+
+  @Override
+  protected String getTemplateFile() {
+    return TEMPLATE_FILE;
+  }
+
+  @Override
+  protected String getMetadataFile() {
+    return METADATA_FILE;
+  }
+
+  @Override
+  public List<String> getEvents() {
+    return Arrays.asList(GITHUB_EVENT_COMMIT_COMMENT, GITHUB_EVENT_ISSUE_COMMENT);
+  }
+
+  @Override
+  protected void preProcessInputData(JsonNode input) {
+    super.preProcessInputData(input);
+    proccessIconURL(input);
+    processUser(input.with(COMMENT_TAG).path(USER_TAG));
+    processUserComment(input);
+    processCommentEntity(input);
+  }
+
+  private void processUserComment(JsonNode rootNode) {
+    ObjectNode commentNode = (ObjectNode) rootNode.path(COMMENT_TAG);
+    SafeString comment = ParserUtils.escapeAndAddLineBreaks(commentNode.path(BODY_TAG).asText());
+    commentNode.put(BODY_TAG, comment.toString());
+  }
+
+  /**
+   * As this class handles issues and commit comments, this method inserts two fields in the final
+   * json:
+   * <ol>
+   * <li><b>Entity</b>: contains the text describing if this event was triggered by an issue or a
+   * commit</li>
+   * <li><b>Entity Url</b>: the url to access this entity</li>
+   * </ol>
+   * @param input Root json node for the webhook event
+   */
+  private void processCommentEntity(JsonNode input) {
+    if (input.has(EVENT_TAG)) {
+      String event = input.get(EVENT_TAG).asText();
```

Comment:
You might change this if clause for:

String event = input.path(EVENT_TAG).asText();